src/content/docs/service-level-management/consume-slm.mdx

```diff
@@ -48,6 +50,8 @@ Depending on what you're trying to achieve, use one the following view modes to
 
 Note that request-based SLOs are determined from SLIs defined as the ratio of the number of good responses to the total number of requests. This means a request-based SLO is met when that ratio meets or exceeds the goal for the SLO compliance period.
 
+Also, SLO compliance results for rolling time windows are more consistent when they include complete weeks. Therefore, SLO periods only include complete weeks. This way, the calculation always includes the same amount of weekends, and any weekly seasonality doesn't impact the results depending on which day of the week you look at SLOs.
```

Comment:
```suggestion
SLO compliance results for rolling time windows are more consistent when they include complete weeks. Therefore, SLO periods only include complete weeks. This way, the calculation always includes the same amount of weekends, and any weekly seasonality doesn't impact the results depending on which day of the week you look at SLOs.
```

Reply:
I try to remove "Also and therefore" as much as possible because I think they can drag down the flow of a doc. But it's personal preference. Feel free to disregard.